rosys/run.py

```diff
@@ -137,15 +131,8 @@ def _kill(proc: subprocess.Popen) -> None:
 
 
 def tear_down() -> None:
-    # stopping process as described in https://www.cloudcity.io/blog/2019/02/27/things-i-wish-they-told-me-about-multiprocessing-in-python/
```

Comment:
I think we should keep the link to the process shutdown tips & tricks

Reply:
Oh, ok. The article does not describe the things we do in our tear_down. At least not explicitly. We can remove it!